src/client/pages/ConnectPayment/components/AdyenCheckout.tsx

```diff
@@ -199,9 +199,9 @@ const createAdyenCheckout = ({
 }: AdyenCheckoutProps) => {
   const locale = match([
     ['sv_SE', 'sv-SE'],
```

Comment:
Should we remove `['sv_SE', 'sv-SE']` since we don't use Adyen in Sweden?

Reply:
Hmm didn't think about that... perhaps removing it will avoid some confusion